src/current/cockroachcloud/connect-to-an-advanced-cluster.md

```diff
@@ -54,7 +54,8 @@ Private connectivity can be configured only after a cluster is created.
     - Set **Target** to **Published service**.
     - Set **Target service** to the value you copied from CockroachDB {{ site.data.products.cloud }} Console. If the endpoint's configured target service does not match, validation will fail.
     - Provide a value for **Endpoint name**. This is not used by CockroachDB {{ site.data.products.cloud }}.
```

Comment:
I think we should replace `This is not used by CockroachDB {{ site.data.products.cloud }}.` with something more explicit (e.g. this endpoint name will be used to connect to your cluster later)

Reply:
There is a previous PR already to address it 
https://github.com/cockroachdb/docs/pull/19419/files